lib/matplotlib/backends/backend_wx.py

```diff
@@ -766,6 +772,8 @@ def _onMouseButton(self, event):
             wx.MOUSE_BTN_LEFT: MouseButton.LEFT,
             wx.MOUSE_BTN_MIDDLE: MouseButton.MIDDLE,
             wx.MOUSE_BTN_RIGHT: MouseButton.RIGHT,
+            wx.MOUSE_BTN_AUX1: MouseButton.FORWARD,
+            wx.MOUSE_BTN_AUX2: MouseButton.BACK,
```

Comment:
```suggestion
            wx.MOUSE_BTN_AUX1: MouseButton.BACK,
            wx.MOUSE_BTN_AUX2: MouseButton.FORWARD,
```
When testing, the assignment seems to be wrong. Please recheck.

Reply:
Yes, I flipped that one. I always have a hard time deciding which is forward and which one is back.